src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -68,11 +68,11 @@ object DepositHandler {
     val tempDir = new File(SwordProps("tempdir"), id)
 
     val result = for {
-      _        <- checkBaseDir()
+      _        <- checkBagStoreBaseDir()
       _        <- extractBag(mimeType)
       bagitDir <- getBagDir(tempDir)
       _        <- checkFetchItemUrls(bagitDir)
-      _        <- checkBagValidity(bagitDir)
+      _        <- checkBagVirtualValidity(bagitDir)
```

Comment:
To make explicit that it is not simply checking that the bag is valid according to the BagIt specs.